rusty_results/tests/exceptions/test_early_return.py

```diff
@@ -0,0 +1,11 @@
+from rusty_results import early_return, Option, Some, Empty
+
+
+def test_early_return():
+    @early_return
+    def __test_it() -> Option[str]:
```

Comment:
I would change the __test_it function to accept a parameter and just use the ~parameter and check if it is empty or Some(10) and we get the 100% coverage back again

Reply:
Just added the pragma to the line that never reaches.